HCaptcha/Classes/HCaptcha.swift

```diff
@@ -92,7 +98,15 @@ public class HCaptcha {
                     infoPlistKey: String?,
                     baseURL: URL?,
                     infoPlistURL: URL?,
-                    size: Size) throws {
+                    size: Size,
+                    rqdata: String?,
+                    sentry: Bool,
+                    apiEndpoint: URL,
```

Comment:
apiEndpoint should be called "jsPath" or similar, based on how you're using it.

Reply:
Done